src/languages/php.coffee

```diff
@@ -44,6 +44,12 @@ module.exports = {
       type: 'string'
       default: ""
       description: "Add rule(s). i.e. line_ending,-full_opening_tag,@PSR2 (PHP-CS-Fixer 2 only)"
+    allow_risky:
```

Comment:
I should have probably seen this documentation from start 👍 . Well everything should be ready now.